gen/calc.py

```diff
@@ -163,6 +163,11 @@ def validate_mesos_log_retention_mb(mesos_log_retention_mb):
     assert int(mesos_log_retention_mb) >= 1024, "Must retain at least 1024 MB of logs"
 
 
+def validate_mesos_container_log_destination(mesos_container_log_destination):
+    assert mesos_container_log_destination in ['journald', 'sandbox', 'both'], \
```

Comment:
I might rename the "destination" in "mesos_container_log_destination" to sink/type/mode, since it's not a concrete destination path, and "destination" makes me think I'm supposed to put a path there.
Maybe also be explicit that 'both' means 'sandbox_and_journald', in case we add a third destination (syslog?) later.